pkg/volume/plugins.go

```diff
@@ -743,3 +744,21 @@ type dummyPluginProber struct{}
 
 func (*dummyPluginProber) Init() error                          { return nil }
 func (*dummyPluginProber) Probe() (bool, []VolumePlugin, error) { return false, nil, nil }
+
+// getBusyboxImage returns multi-arch busybox docker image
+func getBusyBoxImage() string {
+	switch arch := runtime.GOARCH; arch {
+	case "amd64":
+		return "gcr.io/google_containers/busybox"
```

Comment:
Let's use busybox from docker hub directly, infact we are using same in https://github.com/kubernetes/kubernetes/blob/master/test/utils/image/manifest.go#L94

Reply:
Great. Updated. PTAL.